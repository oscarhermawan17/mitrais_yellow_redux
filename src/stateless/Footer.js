import React from 'react';

const Footer = () => 
  (
    <footer>
      Oscar Hermawan <br />
      oscar.hermawan&copy;mitrais.com
    </footer>
  )

    

export default Footer;