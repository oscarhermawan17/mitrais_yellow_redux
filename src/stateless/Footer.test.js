import { shallow } from 'enzyme'
import React from 'react'
import Footer from './Footer'

describe("Footer Stateless", () => {
    it("expect render", () => {
        expect(shallow(<Footer />).length).toEqual(1)
    })

    it("snapshot", () => {
        expect(shallow(<Footer />).length).toMatchSnapshot()
    })
})