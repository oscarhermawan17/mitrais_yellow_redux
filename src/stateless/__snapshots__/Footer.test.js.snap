// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Stateless snapshot 1`] = `ShallowWrapper {}`;
